src/lib/response-triggers/numberconfig.ts

```diff
@@ -106,4 +104,21 @@ export default class NumberConfig extends Trigger {
 
 		return message;
 	}
+
+	private validateConfigActionSet(words: string[]) : string {
+		if (words.length < 4) {
+			return 'You must specify a config name and value to set';
+		}
+
+		const value = Number(words[3]);
+
+		if (isNaN(value)) {
+			return 'Config must be set to a number';
+		}
+
+		if(value < 0) {
+			return 'Config should be greater than 0.';
```

Comment:
"Should be no less than 0" or "should be greater than or equal to 0", surely?

(Also, you've missed the space after `if` repeatedly in this file.)

Reply:
buh, I thought I had fixed the wording. Guess I didn't